src/directives/droppable.directive.ts

```diff
@@ -209,10 +209,10 @@ export class Droppable implements OnInit, OnDestroy {
             this.unbindDragEnterListener();
         }
         if (this.unbindDragOverListener) {
-            this.unbindDragEnterListener();
+            this.unbindDragOverListener();
         }
```

Comment:
This actually took the most amount of time to track down :) Drag over and leave events were getting subscribed to but never being unsubscribed.

Reply:
Good Eye man!